From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Robbie Harwood <rharwood@redhat.com>
Date: Fri, 25 Mar 2022 15:01:54 -0400
Subject: [PATCH] Add -D_GLIBCXX_ASSERTIONS to CPPFLAGS

Signed-off-by: Robbie Harwood <rharwood@redhat.com>
---
 Make.defaults | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

NOTE: This change is already present in pesign 116. The patch body is a no-op so the packaging series continues to apply cleanly after the version bump.

diff --git a/TODO b/TODO
--- a/TODO
+++ b/TODO
@@ -1 +1 @@
-Stuff that needs to get done:
+Stuff that needs to get done:
